Test.Net35/RequestHandlerTest.cs

```diff
@@ -0,0 +1,43 @@
+using NUnit.Framework;
+using Segment.Model;
+using System.Net;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace Segment.Test
+{
+	[TestFixture()]
+	public class RequestHandlerTest
+	{
+		[SetUp]
+		public void Init()
+		{
+			Analytics.Dispose();
```

Comment:
Looking at the other tests in this folder, it looks like we should be logging to the console. Please see example Init() function in ActionTests.cs or ConnectionTests.cs.